src/main/java/seedu/commands/DeleteExpenseCommand.java

```diff
@@ -9,18 +8,12 @@
     private int expenseIndex;
 
     public DeleteExpenseCommand(int expenseIndex) {
-        this.expenseIndex = expenseIndex;
+        this.expenseIndex = expenseIndex - 1;
```

Comment:
maybe expenseNumber (for user input starting from 1) and expenseIndex (for array index) would be clearer